### DISCUSSION TITLE
Recorded Sessions Unavailable but are on server ready to view

### DISCUSSION THREAD

**Discussion by Wolfwood369** [gravitational/teleport#9258] [Q&A]
in both versions 7.1 and now 8.0.1, when I try to view recorded sessions, what I get is a message:

"Recording for this session is not available."

I have searched for an answer to this issue without success. I check the server and the recordings are there. Everything tells me that the session is ready to view. I simply can't play it on the front end.

FYI - I set up teleport with 2 separate containers. One is the frontend that is exposed externally and the other is the authentication server the is behind my firewall. I'm getting the feeling that something about this setup is what is contributing.

**Comment by johanwesterduin** [comment]
Same here

**Comment by Valien** [comment]
Are these SSH sessions that you are trying to replay?

**Comment by Wolfwood369** [comment]
Yes and we got them running. It was as simple as the role. We're still
sorting roles and what they'll have available. This helped a great deal.
